### PR TITLE
Make tag items shorter (32px)

### PR DESCRIPTION
**Pull request by winkerVSbecks**

### CODE REVIEW

File: src/components/tag/TagItem.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const TagItem = styled.div.attrs<TagItemProps>(({ isLoading, children })
   display: inline-block;
   background: ${background.app};
   border-radius: ${spacing.borderRadius.small}px;
-  padding: 5px 10px;
+  padding: 4px 10px 2px 10px;
```
Comment (domyen) [inline src/components/tag/TagItem.tsx:24]: Can we change to `padding: 4px 10px` and adjust the line-height to 20px instead?

Comment (winkerVSbecks) [reply src/components/tag/TagItem.tsx:24]: Do you mean `padding: 5px 10px` cause 4px makes the height 30px. Also, I would recommend `padding: 6px 10px 4px 10px;` for optical correction.

**Comment by domyen** [comment]
I'm suggesting  `padding: 4px 10px` && `line-height: 20px` to yield a 32px
tall tag element.

**Comment by winkerVSbecks** [comment]
@domyen so, `padding: 4px 10px` && `line-height: 20px` yields 30px height. That's why I recommended those other options.

File: src/components/tag/TagItem.tsx (hunk 1 above)
Comment (domyen) [reply src/components/tag/TagItem.tsx:24]: That sounds good re:optical correction

**Review by domyen — APPROVED** [review]

**Comment by winkerVSbecks** [comment]
:rocket: PR was released in [`v5.4.3`](https://github.com/storybookjs/design-system/releases/tag/v5.4.3) :rocket:

**Pull request merged**